tests/unit/test_sql_sensor.py

```diff
@@ -0,0 +1,155 @@
+import unittest
+from unittest import mock
+
+from airflow.exceptions import AirflowException
+from airflow_clickhouse_plugin import ClickHouseSqlSensor
+
+
+class ClickHouseSqlSensorTestCase(unittest.TestCase):
+    @mock.patch(
+        'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor.ClickHouseHook'
+    )
+    def test_clickhouse_sql_sensor_poke(self, mock_hook):
+        op = ClickHouseSqlSensor(task_id='sql_sensor_check', sql='SELECT 1',)
+
+        mock_get_records = mock_hook().get_records
+
+        mock_get_records.return_value = []
+        self.assertFalse(op.poke(None))
+
+        mock_get_records.return_value = [[None]]
+        self.assertFalse(op.poke(None))
+
+        mock_get_records.return_value = [['None']]
+        self.assertTrue(op.poke(None))
+
+        mock_get_records.return_value = [[0.0]]
+        self.assertFalse(op.poke(None))
+
+        mock_get_records.return_value = [[0]]
+        self.assertFalse(op.poke(None))
+
+        mock_get_records.return_value = [['0']]
+        self.assertTrue(op.poke(None))
+
+        mock_get_records.return_value = [['1']]
+        self.assertTrue(op.poke(None))
+
+    @mock.patch(
+        'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor.ClickHouseHook'
+    )
+    def test_clickhouse_sql_sensor_poke_fail_on_empty(self, mock_hook):
+        op = ClickHouseSqlSensor(
+            task_id='sql_sensor_check', sql='SELECT 1', fail_on_empty=True,
```

Comment:
Please rewrite to `sql='SELECT 1 WHERE 0'` for the empty result to be expected. I see that this is a patched unit test but still it would be nice to have a consistent input.